README.md

```diff
@@ -3,7 +3,7 @@
 <div align="center">
   <br/>
 
-  <h3><b> My Todo list</b></h3>
+  <h3><b>Todo list</b></h3>
 
 </div>
 
```

Comment:
### 🔴 Your Pull Request Summary

![image](https://user-images.githubusercontent.com/93039359/228907269-29a73f1e-e32d-468c-8e92-c747c46758ea.png)

### 🟢 Descriptive Pull Request Summary

![image](https://user-images.githubusercontent.com/93039359/228908627-f0b20715-649f-4200-85eb-3e912fa7c82f.png)

- [ ] It would be very great if you be more specific in your pull request summary about your milestone work 👍 You are telling the general things but you should be very specific in explaining the details about this milestone 👍